backend/app/routers/volunteers.py

```
@@ -3,12 +3,49 @@
 from app.models.user import User
-from app.schemas.volunteer import ScheduleCreateRequest, ScheduleCreateResponse
+from app.schemas.volunteer import AppliedPostInfo, ScheduleCreateRequest, ScheduleCreateResponse, ScheduleItem, ScheduleListResponse
+from app.repositories import volunteer_repo
 from app.services import volunteer_service
 
 router = APIRouter()
 
 
+@router.get("/schedules", response_model=ScheduleListResponse)
+async def get_schedules(
+    current_user: User = Depends(get_current_user),
```

Ruff (0.15.11)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)